schema/test/schema.test.ts

```diff
@@ -46,6 +46,9 @@ const basicManifest = Object.freeze({
   post_install_action: {
     type: "settings",
   },
+  constants: {
+    HELLO: "world",
```

Comment:
Do we want to enforce uppercase keys? It might not be possible or easy to do with the schema spec, but it could be helpful if that's a convention we want to follow/encourage.

Reply:
I don't see a reason to enforce that, no. This section is totally up to the developer and doesn't affect us.